homeassistant/components/media_player/samsungtv.py

```diff
@@ -22,7 +22,7 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.util import dt as dt_util
 
-REQUIREMENTS = ['samsungctl[websocket]==0.7.1', 'wakeonlan==1.1.6']
+REQUIREMENTS = ['samsungctl[websocket]', 'wakeonlan==1.1.6']
```

Comment:
We are pinning the releases to make sure that at one point in time the integration worked.